cloud/monitoring/basic-objects/hosts.md

```diff
@@ -16,7 +16,7 @@ To create a host manually:
 * The **Name** field defines the host name that will be used by the Monitoring Engine. The following characters are not authorized: `~!$%^&|'"<>?,()=*{}` and spaces. Using names containing `/` can also have technical impacts on SQL queries, on the interface or cause other compatibility problems with third-party tools. We recommend that you only use `_-.` for host names.
 * The **Alias** field shows the alias of the host.
 * The **Address** field defines the IP address or DNS name of the host. The **Resolve** button enables us to
-  resolve the domain name by questioning the DNS server configured on the SaaS platform.
+  resolve the domain name by questioning the DNS server configured on the SaaS platform. Note that for security reasons, parameters formated as `<protocol>://<ip-address>` are not accepted.
```

Comment:
"parameters" pas sûr de la formulation.
J'aurais plutôt vu un truc du genre "Note that for security reasons, you can't specify address formated as..." 
Not my call really